tests/cypress/support/commands.ts

```diff
@@ -175,10 +173,11 @@ Cypress.Commands.add('checkApplicationStatus', (appNamespace, appName, clusterNa
 });
 
 // Delete the leftover applications
-Cypress.Commands.add('deleteApplicationDeployment', (appNamespace, clusterName='local') => {
+Cypress.Commands.add('deleteApplicationDeployment', (clusterName='local') => {
   cypressLib.burgerMenuToggle();
   cypressLib.accesMenu(clusterName);
-  cy.nameSpaceMenuToggle(appNamespace);
   cy.clickNavMenu(['Workloads', 'Deployments']);
-  cy.deleteAll({fleetCheck: false});
+  // For certain reason deleteAll() is not working
+  // TODO: Investigate and fix it.
```

Comment:
I think this is a speed issue
Adding `cy.wait(500)` , while maintaining `cy.deleteAll(false);` will work here, which makes me think the reason why it does not fail the version in the library is simply because since it is in node modules, probably takes an extra bit of time that is what is needed here. I may be wrong but could be it.

Feel free either to leave your proposal or add the wait (the check of any other visible element probably may work as well)